Nos/Views/AppView.swift

```diff
@@ -160,11 +160,20 @@ struct AppView: View {
     }
 
     private func presentNIP05SheetIfNeeded() async {
-        // Sleep for half a second
-        try? await Task.sleep(nanoseconds: 500_000_000)
         guard let author = currentUser.author, let npub = author.npubString else {
+            // We don't anything about the current user
```

Comment:
I think this is self-explanatory; the code above is clear that we need an author and an npub to perform the rest of the code in this `func`. I'd remove the comment.

```suggestion
```